### ISSUE TITLE
Wetlands show much darker (even black) at zoom level 13 and up.

### ISSUE THREAD

**Issue by tarangill** [openmaptiles/openmaptiles#1624]
<img width="2031" alt="Screenshot 2024-01-27 at 6 34 00 AM" src="https://github.com/openmaptiles/openmaptiles/assets/1241557/a5e75184-eef9-4709-bf70-0231c0cc6ac2">
<img width="1966" alt="Screenshot 2024-01-27 at 6 33 46 AM" src="https://github.com/openmaptiles/openmaptiles/assets/1241557/b3b424a8-0a7c-4ca3-8f91-b78dede79235">

As you can see in the picture above, the wetlands are getting rendered darker at zoom levels 13 and up. Some even show totally black.

**Comment by tarangill** [comment]
Ok,  on more inspection I found that it was due to the patterns layer. I disabled it in my styles and the issue went away. On further inspection I found out my sprites file looks like this:

![sprite@2x](https://github.com/openmaptiles/openmaptiles/assets/1241557/c6e2a138-d7d9-4ae0-9a8b-19f67d5ca1de)

And the area from which the "wetland" sprite is being spliced is black. I am further investigating. I will close this ticket when I find out why, so it can help others facing the same problem.
